src/App.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import Footer from './components/Footer';
+import BooksPage from './Pages/BooksPage';
+import CategoriesPage from './Pages/CategoriesPage';
+import './App.css';
+
+function App() {
+  return (
+    <main>
+      <NavBar />
+      <Routes>
+        <Route path="Books" element={<BooksPage />} />
```

Comment:
- [ ] From the requirements, the books page should be the default view. You can do that by making this component the root path

  ```
  <Route path="/" element={<BooksPage />} />
  ```

Reply:
Thanks a lot for pointing that out!